.circleci/config.yml

```diff
@@ -479,11 +480,12 @@ jobs:
       - restore_cache:
           keys:
             - v5-mail-dependencies-{{ checksum "yarn.lock" }}
-      # If the yarn.lock file is not up-to-date with the package.json file,
-      # using the --frozen-lockfile should fail.
+      - run:
+          name: Check yarn.lock is up to date
+          command: yarn check --integrity
       - run:
           name: Install node dependencies
-          command: yarn install --frozen-lockfile
```

Comment:
`--frozen_lockfile` didn't show correctly if the `yarn.lock` was up to date or not.
@lunika There are other occurences of `--frozen-lockfile` in the config file, I am not sure if we should treat them as well or not.

Reply:
`--frozen-lockfile` is buggy with yarn 1 and is fix in yarn 2 and higher. We should keep it like this IMO

Reply:
I just saw the `--intergity` usage. Why not doing it for the all file ? You think doing it in the first jobs is enough ?

Reply:
I think we should do it for the other occurences as well, but I was not sure.